docs/faq.md

```diff
@@ -172,6 +172,12 @@ Containers:
       --timeout=61s
 ```
 
+## IMDSv2 Support
+
+The driver supports the use of [IMDSv2](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html) (Instance Metadata Service Version 2) for enhanced security when accessing EC2 instance metadata.
```

Comment:
Drop the words "for enhanced security when accessing EC2 instance metadata", as our next paragraph has you completely eliminate most/all those security benefits.

Reply:
Dropped!